Commons/src/main/java/tw/waterball/judgegirl/commons/utils/MapUtils.java

```diff
@@ -0,0 +1,36 @@
+package tw.waterball.judgegirl.commons.utils;
+
+import lombok.AllArgsConstructor;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author - johnny850807@gmail.com (Waterball)
+ */
+public class MapUtils {
+
+    public static <K, V> Map<K, V> zip(K[] keys, V[] values) {
+        Map<K, V> map = new HashMap<>();
+        for (int i = 0; i < keys.length; i++) {
```

Comment:
NIT : keys length might less than values length

Reply:
that's okay, as if are mapping the keys to values, the values that have no keys map to will be ignored.

Reply:
but it would throw IndexOutOfBoundException